src/Microsoft.AspNetCore.Owin/OwinExtensions.cs

```diff
@@ -63,8 +57,43 @@ public static AddMiddleware UseOwin(this IApplicationBuilder builder)
             return add;
         }
 
+        public static AddMiddlewareEx UseOwinEx(this IApplicationBuilder builder)
```

Comment:
I don't like it either, but I didn't come up with a better name... What's your suggestion?